src/app/outbound/mod.rs

```diff
@@ -147,15 +148,15 @@ where
     // over all endpoints returned from the destination service.
     let balancer_layer = svc::layers()
         .push_spawn_ready()
-        .push(resolve::layer(discovery::Resolve::new(resolve)))
+        .push(discover::Layer::new(2, resolve))
         .push(balance::layer(EWMA_DEFAULT_RTT, EWMA_DECAY));
 
     let distributor = endpoint_stack
         .push(
             // Attempt to build a balancer. If the service is
             // unresolvable, fall back to using a router that dispatches
             // request to the application-selected original destination.
-            fallback::layer(balancer_layer, orig_dst_router_layer).on_error::<Unresolvable>(),
+            fallback::layer(balancer_layer, orig_dst_router_layer), // TODO .on_error::<Unresolvable>(),
```

Comment:
i'm guessing this was supposed to be un-commented?

Reply:
yeah, we don't need that anymore. any error that hits this should cause fallback.